app/views/buyers/accounts/show.html.slim

```diff
@@ -63,11 +63,12 @@ h1 data-hook="account-show"
                                  method: :post,
                                  class: 'reject action'
             - if can? :suspend, @account
-              = action_link_to 'Suspend',
-                               suspend_admin_buyers_account_path(@account),
-                               method: :post,
-                               data: { confirm: 'Are you sure?', disable_with: 'suspending…' },
-                               class: 'action suspend'
+              - unless @account.suspended? || @account.scheduled_for_deletion?
```

Comment:
This is correct functionally but a bit hard to read. IMO it is easier:
```ruby
  - if !@account.suspended? && !@account.scheduled_for_deletion?
```

Yet, this still has one problem: If we ever change when it is possible to change to 'suspended' in the states machine (model), then that means that we have to remember to change it here. In fact, even now, there are more reasons why we do not allow to `suspend` and account. So, to do it dynamically and always according to what we write in the model, I suggest this:
```suggestion
              - if @account.can_suspend?
```

Reply:
Just to be clear, `@account.can_suspend?` means "If this object `@account` can be changed to state `suspended`"

Reply:
oh, and even better if we mix it with the condition above instead of inside it.
like: `- if can? :suspend, @account && @account.can_suspend?`